### PR TITLE
LINK-2069 | make maximum attendee capacity field required

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Make maximum attendee capacity field required
Update instruction texts for attendee capacities

## Closes
[LINK-2069](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2069)

## Screenshot
Screenshot 2024-06-12 at 13 54 22

[LINK-2069]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2069?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr410-ui.dev.hel.ninja** :rocket::rocket::rocket:

File: src/domain/app/i18n/sv.json [hunk 1]
```diff
@@ -499,12 +499,18 @@
         "warningNoEmailFound": "Det gick inte att hitta någon e-postadress kopplad till detta evenemang."
       },
       "helperUmbrellaEvent": "Observera! Ett paraplyevenemang måste skapas innan du kan lägga till underevenemang till det.",
-      "infoTextAttendeeCapacity": {
-        "course": "Om antalet deltagare i kursen är begränsat, ange lägsta och högsta antal deltagare här.",
-        "general": "Om antalet deltagare i evenemanget är begränsat, ange lägsta och högsta antal deltagare här.",
-        "volunteering": "Om antalet deltagare i volontärarbetet är begränsat, ange lägsta och högsta antal deltagare här."
+      "infoTextAttendeeCapacity1": "Vänligen ange det maximala antalet deltagare eller det uppskattade antalet deltagare.",
+      "infoTextAttendeeCapacity2": {
+        "course": "Om antalet deltagare i kursen inte är begränsat, vänligen ge en grov uppskattning av det lägsta maximala antalet deltagare (denna information används för statistiska ändamål).",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/sv.json:504]:
```suggestion
        "course": "Om antalet deltagare i kursen inte är begränsat, vänligen ge en grov uppskattning av åtminstone det maximala antalet deltagare (denna information används för statistiska ändamål).",
```
This suggestion is according to ChatGPT.

File: src/domain/app/i18n/sv.json [hunk 2]
```diff
@@ -499,12 +499,18 @@
...
+        "general": "Om antalet deltagare i evenemanget inte är begränsat, vänligen ge en grov uppskattning av det lägsta maximala antalet deltagare (denna information används för statistiska ändamål).",
```
Comment (harriris-vincit) [inline src/domain/app/i18n/sv.json:505]:
```suggestion
        "general": "Om antalet deltagare i evenemanget inte är begränsat, vänligen ge en grov uppskattning av åtminstone det maximala antalet deltagare (denna information används för statistiska ändamål).",
```

File: src/domain/app/i18n/sv.json [hunk 3]
```diff
@@ -499,12 +499,18 @@
...
+        "volunteering": "Om antalet deltagare i volontärarbetet inte är begränsat, vänligen ge en grov uppskattning av det lägsta maximala antalet deltagare (denna information används för statistiska ändamål)."
```
Comment (harriris-vincit) [inline src/domain/app/i18n/sv.json:506]:
```suggestion
        "volunteering": "Om antalet deltagare i volontärarbetet inte är begränsat, vänligen ge en grov uppskattning av åtminstone det maximala antalet deltagare (denna information används för statistiska ändamål)."
```

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr410-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**